tests/test_authentication.py

```diff
@@ -29,14 +30,16 @@ def test_decoding_user_raises_unauthorized_when_invalid_token(user_token: str) -
 
 
 def test_decoding_user_raises_unauthorized_when_invalid_audience(user_token: str) -> None:
-    with pytest.raises(Unauthorized), patch("lbz.jwt_utils.ALLOWED_AUDIENCES", [str(uuid4())]):
+    with pytest.raises(Unauthorized), patch(
+        "lbz.jwt_utils.ALLOWED_AUDIENCES", MockedConfig(str(uuid4()), parser=lambda a: [a])
```

Comment:
It was done this way before as this variable was declared globally. Now, when we base it on configuration values, it shouldn't be needed.

But to remove all these mocks or to make them better, I would need to investigate what they really want to replace 🤔